### PR TITLE
fix: bump kubecost sidecar image

### PR DESCRIPTION
**Pull request by palexster**
**What problem does this PR solve?**:

Let's bump the kiwigrid/sidecar image to the latest one to avoid critical CVEs in auxiliary libraries.

**Which issue(s) does this PR fix?**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

File: services/kubecost/0.37.1/defaults/cm.yaml [hunk 1]
```diff
@@ -75,6 +75,10 @@ data:
           analytics:
             reporting_enabled: false
             check_for_updates: false
+        sidecar:
```
Comment (mhrabovcin) [inline services/kubecost/0.37.1/defaults/cm.yaml:78]: I think you need to create a new service version when changing the default CM.

File: services/centralized-grafana/48.3.1/centralized-grafana.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ spec:
         kind: HelmRepository
         name: mesosphere.github.io-charts-staging
         namespace: kommander-flux
-      version: 48.3.1
+      version: 48.3.2
```
Comment (gracedo) [inline services/centralized-grafana/48.3.1/centralized-grafana.yaml:14]: not sure if this bump was intentional but the latest version is actually 48.3.3 https://github.com/mesosphere/charts/blob/6135b0df4858e916d6bd5705ca63a43c885153bf/staging/kube-prometheus-stack/Chart.yaml#L24

File: services/centralized-kubecost/0.37.1/release/release.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ spec:
         kind: HelmRepository
         name: mesosphere.github.io-charts-stable
         namespace: kommander-flux
-      version: 0.37.1
+      version: 0.37.2
```
Comment (gracedo) [inline services/centralized-kubecost/0.37.1/release/release.yaml:14]: this helm chart version doesn't exist! and I think it's fine not to bump the app version, we just need it to be bumped between dkp versions if there are any changes, and we've already bumped this app from 2.6, so we should be good to keep it at the current app version even if we make config changes

File: services/kube-prometheus-stack/48.3.2/defaults/cm.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: kube-prometheus-stack-48.3.1-d2iq-defaults
+  name: kube-prometheus-stack-48.3.2-d2iq-defaults
```
Comment (gracedo) [inline services/kube-prometheus-stack/48.3.2/defaults/cm.yaml:4]: same comment, we don't need to bump the app version

File: services/kubecost/0.37.2/defaults/cm.yaml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: kubecost-0.37.1-d2iq-defaults
+  name: kubecost-0.37.2-d2iq-defaults
```
Comment (gracedo) [inline services/kubecost/0.37.2/defaults/cm.yaml:4]: and here as well

**Review by gracedo — CHANGES_REQUESTED** [review]
We don't need to bump the app versions since they have been bumped from 2.6

File: services/kubecost/0.37.2/defaults/cm.yaml (hunk 5 above)
Comment (palexster) [reply services/kubecost/0.37.2/defaults/cm.yaml:4]: I thought this was required (reporting the comment), so I did it everywhere. https://github.com/mesosphere/kommander-applications/pull/1785#discussion_r1434016668

Glad to rollback if not needed.

File: services/centralized-grafana/48.3.1/centralized-grafana.yaml (hunk 2 above)
Comment (palexster) [reply services/centralized-grafana/48.3.1/centralized-grafana.yaml]: no it was a mistake, thanks for your review.

**Review by gracedo — APPROVED** [review]
thank you!

File: services/kube-prometheus-stack/48.3.1/defaults/cm.yaml [hunk 6]
```diff
@@ -351,6 +351,9 @@ data:
         path: /dkp/grafana
         pathType: ImplementationSpecific
       sidecar:
+        image:
+          repository: kiwigrid/k8s-sidecar
+          tag: 1.25.3
```
Comment (gracedo) [inline services/kube-prometheus-stack/48.3.1/defaults/cm.yaml:356]:
```suggestion
          tag: 1.25.3
```

**Pull request merged**